app/src/main/java/com/nononsenseapps/feeder/ui/compose/feed/FeedItemCard.kt

```diff
@@ -210,6 +211,12 @@ fun RowScope.FeedItemText(
             }
         }
     val closeMenuText = stringResource(id = R.string.close_menu)
+    // The color of unread items remains unchanged, and the color of read items increases transparency
+    val titleColor = if (item.unread) {
+        Color.Unspecified // Use the default color and do not change the color of unread items
+    } else {
+        LocalContentColor.current.copy(alpha = 0.74f) //Increase transparency of read item color
+    }
```

Comment:
All of this should be moved inside the `FeedListItemTitleTextStyle` function in `Typography.kt`. See how the the font weight uses the unread flag:

```
fontWeight = titleFontWeight(item.unread)
```